install/install_openadapt.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+# Run a command and ensure it did not fail
+RunAndCheck() {
+    res=$($1)
+
+    if [[ -z $res ]] || [[ $res == 0 ]]; then
+        echo "Success: $2 : $res"
+    else
+        echo "Failed: $2 : $res"
+        exit 1
+    fi
+}
+[ -d "puterbot" ] && mv puterbot puterbot-$(date +%Y-%m-%d_%H-%M-%S)
```

Comment:
should be openadapt

Reply:
https://github.com/OpenAdaptAI/install/pull/1 was never merged 😵‍💫

Reply:
@0dm Thank you! I have resolved them :)